pyscriptjs/src/plugins/python/py_markdown.py

```diff
@@ -0,0 +1,31 @@
+from textwrap import dedent
+
+from markdown import markdown
+from pyscript import Plugin, console
+
+console.warning(
+    "WARNING: This plugin is still in a very experimental phase and will likely change"
+    " and potentially break in the future releases. Use it with caution."
+)
+
```

Comment:
again -1 on this. If it's a warning that the user should see, we should display it on the DOM (and add a way to silence it in the config)

Reply:
On this one, we are in strong disagreement here. I'm -100 on raising any warning that is so invasive that we put on the DOM. As a developer user of PyScript I'd be really really frustrated that my users have to see a weird warning that they don't understand and adds -10 value to their experience on my app.

Reply:
our users are the developers, and by having "hidden warnings" we don't do a good service to them.
*Their* users are the final users of the app and yes, the final users don't want to see a warning: that's why there should be an option to silence it, so that *our* user can decide whether to fix the problem or hide the warning.